forte_wrapper/spacy/spacy_processors.py

```diff
@@ -62,18 +108,29 @@ def default_configs(cls):
         """
         This defines a basic config structure for spaCy.
         Returns:
-
+            dictionary with the default config for this processor.
+        Following are the keys for this dictionary:
+            - processors: defines what operations to be done on the sentence,
+                default value is "sentence, tokenize,pos,lemma" which performs
+                all the basic operations.
+            - lang: language model, default is spaCy 'en_core_web_sm' model.
+                The pipeline support spaCy and ScispaCy models.
```

Comment:
You probably need to add `ScispaCy` to the `https://github.com/asyml/forte-wrappers/blob/main/docs/spelling_wordlist.txt` file because we start to enforce documentation check now.